### DISCUSSION TITLE
Note Map need help.

### DISCUSSION THREAD

**Discussion by Heidieker** [TriliumNext/Trilium#4679] [Q&A]
I have some questions. Even with the Trilium demo, I can't understand some things. When I click on the Note Map, I can click on some notes, but not others. How do I link articles together so that they belong together on the Note Map and I can click on them? "~targetNote=" seems to create a link, but then I can't click on the page in the Note Map. Please help me.

**Comment by zadam** [comment]
Sorry for late response.

Hmm, not sure about non-clickable items on the map. Standard way to link them on the note map is to just create a link in the text note (e.g. with CTRL-L). You put a link to note B into note A, that will also show such a link in the note map.

The other way is via [relations](https://github.com/zadam/trilium/wiki/Attributes).
